application/controllers/admin/tokens.php

```diff
@@ -2615,7 +2615,7 @@ protected function getSQLemailstatuscondition()
         if ($request->getPost('bypassbademails') == '1') {
             return "emailstatus = 'OK'";
         } else {
-            return "emailstatus <> 'OptOut'";
+            return "emailstatus <> 'OptOut' OR emailstatus IS NULL";
```

Comment:
although the WHERE condition "emailstatus <> 'OptOut'" intuitively should match everything that is not equal to "OptOut" it actually does not match NULL values in MySQL since it is not NULL-safe.